USAGE.md

```diff
@@ -5,7 +5,7 @@ The purpose of GDL is to store geospatial datasets. This document should provide
 The keywords "must", "must not", "required", "shall", "shall not", "should", "should not", "recommended",  "may", and "optional" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
 ## Prerequisites
-Currently, Data Lake allows read/write access for all users from all AWS accounts (users home accounts) specified in `DATALAKE_USERS_AWS_ACCOUNTS_IDS` during deployment time (see [README](README.md#aws-infrastructure-deployment-cdk-stack)). Please contact the Data Lake product team to add your AWS account to this list.
+Currently, Data Lake allows read/write access for all users authenticated via `DATALAKE_SAML_IDENTITY_PROVIDER_ARN` during deployment time (see [README](README.md#aws-infrastructure-deployment-cdk-stack)).
```

Comment:
Do we need to add some info on how to get added to the AD group, like request access from the LINZ service desk? The name of the group? Not sure that we should put this in a public repo though, as a) it's not relevant to anyone outside of LINZ and b) it could be some minor security risk? Could put it in Confluence?

Reply:
> Data Lake allows read/write access for all users authenticated via `DATALAKE_SAML_IDENTITY_PROVIDER_ARN` during deployment time

This seems more like implementation detail rather than how to use it.

Reply:
> Do we need to add some info on how to get added to the AD group, like request access from the LINZ service desk? The name of the group? Not sure that we should put this in a public repo though, as a) it's not relevant to anyone outside of LINZ and b) it could be some minor security risk? Could put it in Confluence?

Yes, this information is not relevant for public. It should go to Confluence.

Reply:
How to use `DATALAKE_SAML_IDENTITY_PROVIDER_ARN` as deployment configuraion is explained in `README`.